cookbook/deployment/cluster/productionize_cluster.py

```diff
@@ -1,8 +1,195 @@
+
+These directories were designed so that you can modify them using ``kustomize`` to generate a custom Flyte deployment.
+In fact, this is how we create the ``sandbox`` deployment.
+
+Understanding the sandbox deployment will help you to create your own custom deployments.
+
+Understanding the Sandbox
+*************************
+
+The sandbox deployment is managed by a set of kustomize `overlays <https://github.com/kubernetes-sigs/kustomize/blob/master/docs/glossary.md#overlay>`__ that alter the ``base`` configurations to compose the sandbox deployment.
+
+The sandbox overlays live in the `kustomize/overlays/sandbox <https://github.com/flyteorg/flyte/tree/master/kustomize/overlays/sandbox>`__ directory. There are overlays for each component, and a "flyte" overlay that aggregates the components into a single deploy file.
+
+**Component Overlays**
+  For each modified component, there is an kustomize overlay at ``kustomize/overlays/sandbox/{{ component }}``.
+  The overlay will typically reference the ``base`` for that component, and modify it to the needs of the sandbox.
+
+  Using kustomize "patches", we add or override specific configs from the ``base`` resources. For example, in the "console" overlay, we specify a patch in the `kustomization.yaml <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/console/kustomization.yaml>`__. This patch adds memory and cpu limits to the console deployment config.
+
+  Each Flyte component requires at least one configuration file. The configuration files for each component live in the component overlay. For example, the FlyteAdmin config lives at `kustomize/overlays/sandbox/admindeployment/flyteadmin_config.yaml <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/admindeployment/flyteadmin_config.yaml>`__. These files get included as Kubernetes configmaps and mounted into pods.
+
+**Flyte Overlay**
+  The ``flyte`` overlay is meant to aggregate the components into a single deployment file.
+  The `kustomization.yaml overlay <https://github.com/flyteorg/flyte/blob/master/kustomize/overlays/sandbox/flyte/kustomization.yaml>`__ in that directory lists the components to be included in the deploy.
+
+  We run ``kustomize build`` against the ``flyte`` directory to generate the complete `sandbox deployment yaml <https://github.com/flyteorg/flyte/blob/master/deployment/sandbox/flyte_generated.yaml>`__ we used earlier to deploy Flyte sandbox.
+
+Creating Your Own Deployment
+****************************
+
+Before you create a custom deployment, you'll need to `install kustomize <https://github.com/kubernetes-sigs/kustomize#kustomize>`__.
+
+The simplest way to create your own custom deployment is to clone the sandbox deploy and modify it to your liking.
+
+NOTE:
+ This section is getting updated to use the new kustomize installation of Flyte. Link to kustomize
```

Comment:
Is the section still being updated?

Reply:
we can address this after merging, noting it as a TODO

Reply:
Done.